templates/dart/lib/permissions.dart.twig

```diff
@@ -0,0 +1,19 @@
+part of {{ language.params.packageName }};
+
+class Permissions {
+    static String read(String role) {
```

Comment:
Similar comments as Kotlin... Please use `Role` as the parameter type instead of `String`